### PR TITLE
Terrell- worked on delete 

### PR DESCRIPTION
**Pull request by Terrellstewart0**
added delete page and servlet

### CODE REVIEW

**Comment by fmendozaro** [comment]
Resolve the conflicts first

**Comment by Terrellstewart0** [comment]
conflicts are resolved please look again

File: src/main/java/com/codeup/adlister/controllers/DeleteAdServlet.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+public class DeleteAdServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+
+
+        long id = Long.parseLong(request.getParameter("id"));
+        Ad ad = DaoFactory.getAdsDao().getAd(id);
+        DaoFactory.getAdsDao().deleteAd(ad);
+
+        if (request.getSession().getAttribute("user") != null) {
```
Comment (MontealegreLuis) [inline src/main/java/com/codeup/adlister/controllers/DeleteAdServlet.java:26]: This conditional should be before line 22. Checking if the user is logged in should be the first action in this servlet. The redirect wouldn't be in the `else` section, but after the ad has been successfully deleted.

File: src/main/java/com/codeup/adlister/dao/MySQLAdsDao.java [hunk 2]
```diff
@@ -112,6 +115,47 @@ public Ad getAd(long id) {
...
+
+        } catch (SQLException e) {
+            throw new RuntimeException("Error updating Profile information", e);
+        }
+
+    }
+
+
+
+//    @Override
```
Comment (MontealegreLuis) [inline src/main/java/com/codeup/adlister/dao/MySQLAdsDao.java:140]: This is more an observation. It is always better to remove code than commenting it out. If you ever need this code again you can always get it back from a commit using git.

**Review by MontealegreLuis — APPROVED** [review]
It looks good. Thank you!

**Pull request merged**